src/com/facebook/buck/ide/intellij/IjProjectCommandHelper.java

```diff
@@ -304,6 +304,7 @@ private void copyGeneratedGoCode(
         DefaultSourcePathResolver.from(new SourcePathRuleFinder(result.getActionGraphBuilder()));
     for (BuildTargetSourcePath sourcePath : generatedPackages.keySet()) {
       Path desiredPath = vendorPath.resolve(generatedPackages.get(sourcePath));
+      fs.deleteRecursivelyIfExists(desiredPath);
```

Comment:
This will not handle sub-packages well.
For example if we have:
github.com/foo/bar
github.com/foo/bar/A
and process the later first then contents of A will be removed while processing root package of the repo.

Let's make sure we cleanup only contents of the current package.